geometry/optimization/graph_of_convex_sets.h

```diff
@@ -105,19 +113,50 @@ graph, such as the shortest path problem where each visit to a vertex also
 corresponds to selecting an element from the convex set subject to the costs
 and constraints.  Behind the scenes, we construct efficient mixed-integer
 convex transcriptions of the graph problem using MathematicalProgram.
+However, we provide the option to solve an often tight convex relaxation of the
+problem with GraphOfConvexSetsOptions::convex_relaxation and employ a cheap
+rounding stage to find a feasible solution to the original problem.
 
 Design note: This class avoids providing any direct access to the
 MathematicalProgram that it constructs nor to the decision variables /
 constraints.  The users should be able to write constraints against
 "placeholder" decision variables on the vertices and edges, but these get
 translated in non-trivial ways to the underlying program.
 
+**Advanced Usage: Guiding Non-convex Optimization with the GraphOfConvexSets**
+The convex relaxation of the GCS problem has two components:
```

Comment:
This part can be better explained? I would not say that the convex relaxation has two parts, one of which is the convex relaxation.